app/assets/javascripts/discourse/mixins/pan-events.js.es6

```diff
@@ -7,11 +11,23 @@ export default Ember.Mixin.create({
     this._super();
 
     if (this.site.mobileView) {
-      this.$()
-        .on("pointerdown", e => this._panStart(e))
-        .on("pointermove", e => this._panMove(e))
-        .on("pointerup", e => this._panMove(e))
-        .on("pointercancel", e => this._panMove(e));
+      if ("onpointerdown" in document.documentElement) {
```

Comment:
This could use `this.capabilities.touch` too right? If not, should it be added to capabilities?